front-end/src/Components/Menu.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+import { device } from "../utils/device";
+
+const MenuStyled = styled.div`
```

Comment:
Can we create sub modules based on components functionality ? I am afraid that we end up with  all of the components in the same place.